main.py

```diff
@@ -1,2 +1,9 @@
+from lesson8 import task8
+
+task8.oops()
```

Comment:
Две пустых строки после импортов
```suggestion

task8.oops()
```